### ISSUE TITLE
logarithmic color_continuous_scale

### ISSUE THREAD

**Issue by eisenlohr** [plotly/plotly.py#3616]
It would be helpful to have an option (named, for instance, `log_color`) that allows using color maps on logarithmic scales, similar to `log_x`, `log_y`, `log_z`. This would essentially rescale the standard linear color range to the corresponding log values, i.e. coloring of `v` would be linear in `log(v)`.

**Comment by cemde** [comment]
is this related to #3735 ?

**Comment by eisenlohr** [comment]
> is this related to #3735 ?

Yes, seems to be the same issue. What is missing is a simple way to have the color scale be equidistant on a log (in contrast to linear) scale.

Let's say the color range is between values A and B.
Linear example: value (A+B)/2 is colored using the midpoint of the color scale.
Log example: value sqrt(A*B) is colored using the midpoint of the color scale
